docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -2898,6 +2898,50 @@ Result:
 └─────────────────────┴─────────────────────┴──────────┘
 ```
 
+## now64InBlock {#now64InBlock}
+
+Returns the current date and time with nanosecond precision at the moment each block of data is processed. Unlike the now64 function, which returns a constant value for the entire query, now64InBlock produces a unique timestamp per block, making it useful for scenarios where time needs to reflect the actual moment of data processing during long-running queries.
```

Comment:
For consistency, could you please make this text sound like the docs for function `now` above?